homeworks/ekaterina.sharova_rogerosha/PokemonProject/css/login.css

```diff
@@ -0,0 +1,199 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.upper-header {
+  background-color: #333;
+  display: grid;
+  align-items: center;
+  grid-template-columns: 1fr auto;
+  padding: 10px;
+  color: #333;
+  gap: 50px;
+  width: 100%;
+  justify-content: space-between;
+}
+
+.ma-logo {
+  display: flex;
+  align-items: center;
+}
+
+.MAlogo {
+  margin-left: 53px;
+}
+
+.navigation {
+  text-align: right;
+  display: flex;
+  align-items: center;
+}
+
+.navigation ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.navigation-item {
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  color: #FFF;
+  line-height: 19px;
+  margin-right: 50px;
+}
+
+.pokemons {
+  text-decoration: none;
+}
+
+.about {
+  text-decoration: none;
+}
+
+.content {
+  max-width: 1280px;
+  padding-inline: 352px;
+}
+
+.headline {
+  color: #221F1F;
+  font-weight: 700;
+  font-size: 48px;
+  line-height: 32px;
+  margin-top: 80px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.authorization-container {
+  text-align: center;
+}
+
+.enter-content {
+  align-items: center;
+}
+
+.email {
+  font-size: 22px;
+  font-weight: 600;
+}
+
+.email-check {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 22px;
+  margin-top: 60px;
+  margin-bottom: 40px;
+  font-weight: 600;
+}
+
+.input-email {
+  width: 100%;
+  border-radius: 6px;
+  border: 1px solid #A9A9A9;
+  padding-block: 30px;
+  padding-inline: 20px;
+}
+
+.password {
+  font-size: 22px;
+}
+
+.password-check {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 22px;
+  margin-block: 40px;
+  font-weight: 600;
+}
+
+.input-password {
+  width: 100%;
+  border-radius: 6px;
+  border: 1px solid #A9A9A9;
+  padding-block: 30px;
+  padding-inline: 20px;
+}
+
+.robot-check {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: flex-end;
+  margin-block: 40px;
+}
+
+.label-checkbox {
+  display: flex;
+  gap: 12px;
+  align-items: center;
+  cursor: pointer;
+}
+
+.real-checkbox {
+  width: 0;
+  height: 0;
+  opacity: 0;
+  position: absolute;
+  z-index: -1;
+}
+
+.custom-checkbox {
+  position: relative;
+  display: inline-block;
+  width: 20px;
+  height: 20px;
+  border-radius: 6px;
+  border: 1px solid #EF4934;
+  background-color: #FFF;
+  cursor: pointer;
+}
+
+.custom-checkbox::before {
+  content: " ";
+  display: inline-block;
+  width: 13px;
+  height: 13px;
+  background-image: url("/homeworks/ekaterina.sharova_rogerosha/PokemonProject/images/checkmark.png");
+  background-size: contain;
+  position: absolute;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%) scale(0);
+  transition: .2s ease-in;
+}
+
+.real-checkbox:checked + .custom-checkbox::before {
+  transform: translate(-50%, -50%) scale(1);
+}
+
```

Comment:
I don't see a checked state 
<img width="910" alt="Screenshot 2023-10-28 at 11 38 05" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/118203044/a333005e-40fa-43d1-9898-7ff336ec38ee">

Reply:
Please add a margin-bottom, like on the layout

Reply:
> I don't see a checked state <img alt="Screenshot 2023-10-28 at 11 38 05" width="910" src="https://user-images.githubusercontent.com/118203044/278801319-a333005e-40fa-43d1-9898-7ff336ec38ee.png">

I can't fix it :( 
Just don't know the way to make it work

Reply:
You can either follow this [example](https://codepen.io/nikkz/pen/BzVBJo) or use the default checkbox. Both variants are accepted.

Reply:
Now it works as it must work


![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/71977925/995328e6-7cb9-4992-8f34-c1c559ef1f9f)


And I also added extra task with filters for the star


![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/71977925/2696a535-4c17-4075-b63c-55bd374b1c3c)